### ISSUE TITLE
It does not convert numbered items

### ISSUE THREAD

**Issue by robo3945** [ankushshah89/python-docx2txt#12]
Your code does not convert numbered items in lists. 

An example:

1. bla bla bla bla
2. bla bla bla bla

Are converted in:

bla bla bla bla
bla bla bla bla

**Comment by Deep12d** [comment]
Hi,
I too faced the same issues mentioned by robo3945.
Also I we should add the code to extract the endnotes.xml & footnotes.xml . Similar to the header and footer*.xml

**Comment by ankushshah89** [comment]
I will try to work on this feature but feel free to send a pull request too.

**Comment by HubCatz202** [comment]
Yes this would be super, super useful if we could extract bullets/numbered lists specifically!

**Comment by biswajithgopinathan** [comment]
Hi..Somebody have any update on this issue. Still I am not able to extract bullets/numbered lists

**Comment by MostafaRabia** [comment]
Hi...?
Is anybody here...?

**Comment by ShayHill** [comment]
I have a fork that converts bullets and numbers in docx to plain text.

[https://github.com/ShayHill/docx2python](https://github.com/ShayHill/docx2python)

To get something close to python-docx2txt output:

```
pip install docx2python
text = docx2python('path/to/file.docx').text
```

**Comment by MostafaRabia** [comment]
@ShayHill 
very good, thank you

**Comment by MostafaRabia** [comment]
@ShayHill 
Is there is option to not to convert hyper links to a tag? I just need the text.

**Comment by ShayHill** [comment]
There is not, though this might be accomplished post-conversion with re.sub:

`re.sub(r'<a href=".*?">(.*?)<a/>', r"\1", text)`

**Comment by MostafaRabia** [comment]
very good, thanks
but it's </a> not <a/> 😅😅
